android/src/main/java/com/amplitude/android/DefaultTrackingOptions.kt

```diff
@@ -0,0 +1,24 @@
+package com.amplitude.android
+
+class DefaultTrackingOptions(
+    var trackingSessionEvents: Boolean = true,
+    var trackingAppLifecycleEvents: Boolean = false,
+    var trackingDeepLinks: Boolean = false,
+    var trackingScreenViews: Boolean = false
```

Comment:
I think we should follow the [same naming convent as Browser](https://github.com/amplitude/Amplitude-TypeScript/blob/7d0281ceefd6087b27f854bd3865a49eaf8f745a/packages/analytics-types/src/config/browser.ts#L30-L36)

```suggestion
    var sessions: Boolean = true,
    var appLifecycle: Boolean = false,
    var deepLinks: Boolean = false,
    var screenViews: Boolean = false
```

Reply:
@justin-fiedler , good call! I can update, will use `appLifecycles` instead of `appLifecycle` for consistency.